src/store/images.js

```diff
@@ -0,0 +1,23 @@
+import { applyReducerHash } from '@redhat-cloud-services/frontend-components-utilities/ReducerRegistry';
+import { LOAD_IMAGES } from './action-types';
+
+const initialState = {};
+
+const loadImagesPending = (state) => ({
+  ...state,
+  data: null,
+  status: 'PENDING',
+});
+const loadImagesFulfilled = (state, { payload }) => ({
+  ...state,
+  data: payload,
+  status: 'DONE',
+});
+
+export default applyReducerHash(
+  {
+    [`${LOAD_IMAGES}_PENDING`]: loadImagesPending,
+    [`${LOAD_IMAGES}_FULFILLED`]: loadImagesFulfilled,
```

Comment:
Error state can be caught here
```suggestion
    [`${LOAD_IMAGES}_FULFILLED`]: loadImagesFulfilled,
    [`${LOAD_IMAGES}_ERROR`]: () => ({ loaded: true, hasError: true }),
```